ui/overlay.reel/overlay.js

```diff
@@ -171,6 +154,8 @@ exports.Overlay = Component.specialize( /** @lends Overlay.prototype # */ {
 
                 _window = this.element.ownerDocument.defaultView;
                 _window.addEventListener("resize", this);
+
+                this._pressComposer = new PressComposer();
```

Comment:
@benoitmarchant I don't think so, given that it's an overlay we need to be able to know if an user click outside an overlay in order to hide that overlay.